### ISSUE TITLE
Did the namespace not get bumped for v8?

### ISSUE THREAD

**Issue by lgritz** [fmtlib/fmt#2374]
For example, https://github.com/fmtlib/fmt/blob/master/include/fmt/core.h#L231

Should that say v8?

**Comment by SoapGentoo** [comment]
Could this potentially be related to https://bugs.gentoo.org/797406

```
In file included from src/common/common.h:39,
                 from src/common/common_pch.h:1,
                 from src/extract/mkvextract.cpp:13:
/usr/include/fmt/format.h:2752:64: error: ‘udl_arg’ in namespace ‘fmt::v7::detail’ does not name a template type
 2752 | stexpr auto operator"" _a(const char* s, size_t) -> detail::udl_arg<char> {
      |                                                             ^~~~~~~
```

**Comment by zx2c4** [comment]
Gentoo bug is unrelated, but is fixed by https://github.com/fmtlib/fmt/pull/2376

**Comment by vitaut** [comment]
Thanks for reporting. This is a KP that will be fixed in the upcoming patch release.

**Comment by vitaut** [comment]
Fixed in 8.0.1: https://github.com/fmtlib/fmt/releases/tag/8.0.1. Thanks for reporting.
